docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,356 @@
+# Installing With SR-IOV Compute Nodes
+
+## Table of Contents
+
+- [Pre-Install Requirements](#pre-install-requirements)
+- [Preparing your OpenShift Cluster to use SR-IOV Networks](#preparing-your-openshift-cluster-to-use-sr-iov-networks)
+- [Creating SR-IOV Networks for Compute Machines](#creating-sr-iov-networks-for-compute-machines)
+- [Creating SR-IOV Compute Nodes in IPI](#creating-sr-iov-compute-nodes-in-ipi)
+- [Creating SR-IOV Compute Nodes in UPI](#creating-sr-iov-compute-nodes-in-upi)
+
+## Pre-Install Requirements
+
+Using single root I/O virtualization (SR-IOV) networking as an additional network in OpenShift can be beneficial for applications
+that require high bandwidth and low latency. If you are planning an OpenStack deployment with SR-IOV NICs enabled, refer to [the OSP 16.1 installation documentation](osp-sriov-install). Before installing, make sure that the network interface cards your OpenStack nodes are using [are supported](supported-nics) for use with SR-IOV
```

Comment:
No need to sale the feature here. The user is already coming to look how to do it, so they are probably using it.